src/styles/blocks/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $main-card-width + 100px;
```

Comment:
Using arithmetic operations in property values can lead to unexpected results if the variable changes. Consider using a predefined variable for the width or ensure that the arithmetic operation is intentional and documented.